packages/cactus-plugin-ledger-connector-iroha/src/main/typescript/prometheus-exporter/metrics.ts

```diff
@@ -0,0 +1,9 @@
+import { Gauge } from "prom-client";
+
+export const K_CACTUS_IROHA_TOTAL_TX_COUNT = "cactus_iroha_total_tx_count";
+
+export const totalTxCount = new Gauge({
```

Comment:
@hxlaf Please take a look at the changes made in this PR (as of today on the main branch) and apply the same fixes to the prometheus related code for the iroha connector so that it is consistent.
https://github.com/hyperledger/cactus/pull/1203

Reply:
Thank you for the reminder! Read and fixed.